src/components/AddressFormFields.tsx

```diff
@@ -1,172 +1,149 @@
-import React, { FC } from "react";
-import { Form, Col } from "react-bootstrap";
-
-interface AddressFormFieldsProps {
-  setName: (name: string) => void;
-  setEmail: (email: string) => void;
-  setAddress: (address: string) => void;
-  setAddress2: (address2: string) => void;
-  setCity: (city: string) => void;
-  setState: (state: string) => void;
-  setZip: (zip: string) => void;
-  loading: boolean;
-}
-
-const AddressFormFields: FC<AddressFormFieldsProps> = (
-  props: AddressFormFieldsProps
-) => {
+import React, { useState, useEffect } from "react";
+import { Form } from "react-bootstrap";
+import GooglePlacesAutocomplete from 'react-google-places-autocomplete';
+import Select from 'react-select';
+import {getCountries, getStates} from 'country-state-picker'
+import { getCountryFromName } from 'iso-country-utils';
+
+const AddressFormFields = (props) => {
```

Comment:
can you type these props please - that's what the interface above is for, so that callsites know which props to pass in, and which are required

Reply:
will do